vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java

```diff
@@ -223,22 +171,21 @@ public void log(String message) {
     // ex for dialog msg dialog.addMessage(Resources.getString("GameRefresher.counters_refreshed_test", updatedCount));
     // Log to chatter
     GameModule.getGameModule().warn(message);
-    //msg.append(new Chatter.DisplayText(chatter, message));
     logger.info(message);
   }
 
   public List<GamePiece>  getCurrentGameRefresherPieces() {
+    decks = new ArrayList<>();
```

Comment:
I remember that Decks carry around a lot of references to a lot of stuff. Probably best to set `decks=null;` once the whole operation is done so that these don't last through a Close Game or whatever.

Reply:
Good point